src/NuGet.Core/NuGet.Configuration/PackageSourceMapping/PackageSourceMapping.cs

```diff
@@ -77,14 +84,7 @@ public static PackageSourceMapping GetPackageSourceMapping(ISettings settings)
 
         private SearchTree GetSearchTree()
         {
-            SearchTree patternsLookup = null;
-
-            if (IsEnabled)
-            {
-                patternsLookup = new SearchTree(this);
-            }
-
-            return patternsLookup;
+            return new SearchTree(this);
```

Comment:
nit: now that this method is only 1 line, it can probably be inlined.

Reply:
I thought about that but believe the callstacks would be easier to look at this way.

Reply:
The stack would just show the `SearchTree` constructor anyways, as that's where the work is actually being done.

Reply:
True... well, in future PR I might remove it if I don't extract it into an interface to make testing easier.